test/modules/op/bmm.py

```diff
@@ -27,3 +29,39 @@ def forward(self, x, y):
 
     def get_example_inputs(self):
         return (torch.randn(2, 4, 5), torch.randn(2, 5, 3)), {}
+
+
+@use_onert
+class SimpleBatchMatMulConstLhs1B(TestModuleBase):
+    def __init__(self):
+        super().__init__()
+        self.const_lhs = torch.randn(1, 4, 5)
+
+    def forward(self, rhs):
+        z = torch.bmm(self.const_lhs, rhs)
+        return z
+
+    def get_example_inputs(self):
+        return (torch.randn(1, 5, 3),), {}
+
+    def get_compile_config(self):
+        return CompileConfigV1(convert_lhs_const_mm_to_fc=True)
```

Comment:
<img width="284" height="816" alt="image" src="https://github.com/user-attachments/assets/5224ae3f-92d0-4eae-9431-8519248a3bcf" />

Reply:
fc output shape (`4x3`) is right? As I understand, it should be `3x4`
```
fc_input  = {     n_batch, in_feature } = {3x5}
fc_weight = { out_feature, in_feature } = {4x5}
fc_output = {     n_batch, out_feature} = {3x4}
```

Reply:
@glistening 

Above case, FC's input = {4x5} and weight = {3x5}, where `weight` is given as actual input to the graph.